### PR TITLE
Add Teleport node Pod for maneki team

### PR DESCRIPTION
**Pull request by kfyharukz**
Signed-off-by: Kazuhito MATSUDA <kazuhito.matuda@gmail.com>

### CODE REVIEW

File: teleport/base/apps/alertmanager.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 10000
+      volumes:
+      - name: teleport-general-secret
+        secret:
+          secretName: teleport-general-secret-20210310
+      - name: teleport-storage
+        emptyDir: {}
+      serviceAccountName: teleport
```
Comment (morimoto-cybozu) [inline teleport/base/apps/alertmanager.yaml:64]: This file is not referred by kustomization.yaml.
It seems to me that this was revived accidentally.

File: secrets/base/teleport/teleport.yaml [hunk 2]
```diff
@@ -2,9 +2,9 @@ teleport:
   data_dir: /var/lib/teleport
   auth_servers:
     - teleport.gcp0.dev-ne.co:443
-  auth_token: teleport-app-token
+  auth_token: teleport-general-token
   log:
     output: stderr
-    severity: INFO
+    severity: DEBUG
```
Comment (morimoto-cybozu) [inline secrets/base/teleport/teleport.yaml:8]: I'm worried that this could produce too many logs.

File: teleport/base/nodes/maneki.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        resources:
+          requests:
+            memory: 256Mi
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 10000
+      volumes:
+      - name: teleport-node-secret
+        secret:
+          secretName: teleport-node-secret-20210309
```
Comment (morimoto-cybozu) [inline teleport/base/nodes/maneki.yaml:62]:
```suggestion
          secretName: teleport-general-secret-20210309
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: secrets/base/teleport/teleport.yaml (hunk 2 above)
Comment (kfyharukz) [reply secrets/base/teleport/teleport.yaml:8]: This configuration only for dctest, so I think DEBUG logs are useful if something wrong in CI.

File: teleport/base/nodes/maneki.yaml (hunk 3 above)
Comment (kfyharukz) [reply teleport/base/nodes/maneki.yaml:62]: 🙏

File: teleport/base/apps/alertmanager.yaml (hunk 1 above)
Comment (kfyharukz) [reply teleport/base/apps/alertmanager.yaml]: 🙏

**Review by morimoto-cybozu — APPROVED** [review]
LGTM
The branch name is confusing:-D

**Review by morimoto-cybozu — APPROVED** [review]

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**